common/src/main/scala/uk/ac/wellcome/models/Concept.scala

```diff
@@ -1,6 +1,88 @@
 package uk.ac.wellcome.models
 
+sealed trait AbstractConcept {
```

Comment:
As discusssed, we should probably override Circe's default discrimator value (the class name) to isolate our JSON format changing when we change class names. If this is safely or easily possible in Circe.

Reply:
Uhm you can configure the name of the field (as in it can be called anything, not necessarily `type`) but you can't configure the value stored in there, I don't think